Source/Graphics/UnitTest/GraphicsTest.cpp

```diff
@@ -0,0 +1,59 @@
+/*
+* @file     GraphicsTest.cpp
+* @author   Marek Cernak
+* @date     9/16/2018
+*
+* @brief Graphics module tests
+*/
+
+#include <gtest/gtest.h>
+#include <memory>
+#include "../RenderWindow.hpp"
+
+namespace gdt
+{
+    namespace test
+    {
+
+        class GraphicsTestBase : public ::testing::Test {
+
+        protected:
+
+            /*
+            * Define ctor for to initialize for whole duration of test.
+            */
+            // ~GraphicsTestBase(){}
```

Comment:
Why commented code?

Reply:
Not needed now, using it just for example. Default ctor and dtor are enough in this class as is right now.